agent/virus_total_agent.py

```diff
@@ -32,6 +33,7 @@ def __init__(
         """
         super().__init__(agent_definition, agent_settings)
         self.api_key = self.args.get("api_key")
+        self.whitelist_types = self.args.get("whitelist_types", [])
```

Comment:
The argument should be added to ostorlab.yaml

Reply:
This is part of the other PR, just pushed by mistake.